webpack.config.js

```diff
@@ -38,10 +38,12 @@ module.exports = (env) => {
     module: {
       rules: [
         {
+          exclude: /(node_modules)/,
           test: /\.js$/,
           loader: 'babel-loader',
           query: {
-            presets: ['@babel/preset-env']
+            presets: ['@babel/preset-env'],
+            plugins: ['@babel/plugin-transform-runtime']
```

Comment:
What does this do that the poly-fill wansn't?

Reply:
https://babeljs.io/docs/en/babel-plugin-transform-runtime#why

Main reason is this will not pollute the global namespace. 
> Another purpose of this transformer is to create a sandboxed environment for your code. If  you directly import `core-js` or `@babel/polyfill` and the built-ins it provides such as `Promise`, `Set` and `Map`, those will pollute the global scope. While this might be ok for an app or a command line tool, it becomes a problem if your code is a library which you intend to publish for others to use or if you can't exactly control the environment in which your code will run.